setup.py

```diff
@@ -31,6 +31,7 @@
         "pyomo>=5.7,<6.6.2",
         "linopy>=0.2.1",
         "matplotlib",
+        "geopandas",
```

Comment:
```suggestion
        "geopandas>=0.9",
```
add version constraint here as well?